vortex-error/src/lib.rs

```diff
@@ -61,6 +63,12 @@ pub enum VortexError {
         arrow_schema::ArrowError,
     ),
     #[error(transparent)]
+    FlatBuffersError(
```

Comment:
These do not have an error, we should write From ourselves and capture backtrace